src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -669,6 +669,7 @@ void GNAPlugin::LoadNetwork(const CNNNetwork& _network) {
             precisions_array{{ngraph::element::f16, ngraph::element::f32}});
         manager.register_pass<ngraph::pass::ConvertMVN1ToMVN6>();
         manager.register_pass<ov::intel_gna::pass::DecomposeMVN>();
+        manager.register_pass<ov::intel_gna::pass::InsertCopyBeforeBroadcastLayer>();
```

Comment:
Why it is located before `CommonOptimizations`?

Reply:
> it was done, because `CommonOptimizations` contains `ConvertBroadcastToTiles` transformation the same as `ConvertOpSet1ToLegacy`.
> 
> ```
>     auto decomp = manager.register_pass<GraphRewrite>();
> ...
>     ADD_MATCHER(decomp, ConvertBroadcastToTiles)
> ```

Before CommonOptimization some constant subgraphs can be unfolded yet and your transformation will fail. 

Also we can have the same situation with other layers between `Parameter` and `Result`. They can be removed by `NopElimination` for example. Maybe there is a sense to think about common approach?

Reply:
I have move transformation to:

        manager.register_pass<ov::intel_gna::pass::InsertCopyBeforeBroadcastLayer>();
        manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
        manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
        manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
It has to be executed before ConvertOpSet1ToLegacy

Reply:
> I have move transformation to:
> 
> ```
>     manager.register_pass<ov::intel_gna::pass::InsertCopyBeforeBroadcastLayer>();
>     manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
>     manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
>     manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
> ```
> 
> It has to be executed before ConvertOpSet1ToLegacy

Do we need to handle "empty" Tile as well?

Reply:
> Do we need to handle "empty" Tile as well?

If I understood you correctly, you are asking if this transformation will handle the situation that in the network will by just `Tile` with `repeats` equal to 1 e.g. {1,1,1} (without any broadcast).

The answer is, No. There will be still problem for tile there, due the fact transformation handle only "empty" broadcast.